### ISSUE TITLE
The llama-2-7b model can't quant in this code

### ISSUE THREAD

**Issue by Hzqskywkr** [OpenGVLab/OmniQuant#93]
Dear authors, we have tried llama-2-7b omniquant in W4A4 following the given script. It is not stable nor successful in training as follows, the loss in the first epoch is NAN.  The training scheme is the same. Are there any solutions to fix it?
 We set the epoch=1,
CUDA_VISIBLE_DEVICES=0 python main.py \
--model hzq/llama/llama-2-7b-hf --eval_ppl \
--epochs 1 --output_dir ./log/Llama-2-7b-chat-w4a4 \
--wbits 4 --abits 4 --lwc --let \
--let_lr 1e-3 --alpha 0.75
![Screenshot from 2024-09-06 19-14-35](https://github.com/user-attachments/assets/6c9a5606-890d-4094-9cbd-c1f93d6190c6)

**Comment by ZijianYY** [comment]
same problem

**Comment by ChenMnZ** [comment]
Please try more epochs. Just set `--epochs` as larger numbers.

**Comment by luo13zhi** [comment]
Loss is NAN when I quantize layer 30, even when  I set epoch larger like 20 or 50, and lower the learning rate like 5e-7. This is my script:
`CUDA_VISIBLE_DEVICES=0 python main.py \
--model /path/to/model/Llama-2-7b-hf --eval_ppl \
--epochs 50 --output_dir ./log/Llama-2-7b-w4a4 \
--wbits 4 --abits 4 --lwc --let  \
--let_lr 5e-7 --lwc_lr 5e-7 --alpha 0.75`
#


> Dear authors, we have tried llama-2-7b omniquant in W4A4 following the given script. It is not stable nor successful in training as follows, the loss in the first epoch is NAN. The training scheme is the same. Are there any solutions to fix it? We set the epoch=1, CUDA_VISIBLE_DEVICES=0 python main.py --model hzq/llama/llama-2-7b-hf --eval_ppl --epochs 1 --output_dir ./log/Llama-2-7b-chat-w4a4 --wbits 4 --abits 4 --lwc --let --let_lr 1e-3 --alpha 0.75 ![Screenshot from 2024-09-06 19-14-35](https://github.com/user-attachments/assets/6c9a5606-890d-4094-9cbd-c1f93d6190c6)

The problem I occured is definitely like this.
